public/index.html

```diff
@@ -9,7 +9,16 @@
 
 		<script defer src="https://use.fontawesome.com/releases/v5.0.13/js/all.js" integrity="sha384-xymdQtn1n3lH2wcu0qhcdaOpQwyoarkgLVxC/wZ5q7h9gHtxICrpcaSUfygqZGOe" crossorigin="anonymous"></script>
 		<link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
-    <title>Drop App</title>
+    <title>Boxwise</title>
+
+		<link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon.png">
```

Comment:
Not a big deal right now, but you've got tabs in here. Two spaces is pretty standard now for HTML, CSS, and JS. You can configure your editor to use two spaces instead of tabs.